hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/impl/AbstractTestChunkManager.java

```diff
@@ -128,6 +135,47 @@ protected void checkChunkFileCount(int expected) {
     assertEquals(expected, files.length);
   }
 
+  /**
+   * Helper class to check if a file is in use.
+   */
+  public static class FuserCheck {
+    public static boolean isFileNotInUse(String filePath) {
+      try {
+        Process process = new ProcessBuilder("fuser", filePath).start();
+        BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+        return reader.readLine() == null;  // If fuser returns no output, the file is not in use
+      } catch (IOException e) {
+        LOG.warn("Failed to check if file is in use: {}", filePath, e);
+        return false;  // On failure, assume the file is in use
+      }
```

Comment:
Should we add a finally block to cleanup, but we would need to define process outside the try block then:
`finally {
            if (process != null) {
                process.destroy();
            }
        }`